jina/flow/base.py

```diff
@@ -837,11 +874,23 @@ def _get_routing_table(self) -> RoutingTable:
                     pod.head_args.hosts_in_connect.append(
                         graph._get_target_pod(start).full_out_address
                     )
-
-                    graph.add_edge(start, end, True)
+                    if end not in graph.pods:
+                        end = end + '_head'
+                    graph.add_edge(start, end, False)
```

Comment:
Why is this `False`? I think this is the reason for `tests/distributed/test_local_flow_use_remote_executor/test_integration.py::test_local_flow_use_external_executor` not passing

Reply:
Thanks, I changed it back to True